ext/standard/uniqid.c

```diff
@@ -71,7 +72,14 @@ PHP_FUNCTION(uniqid)
 	 * digits for usecs.
 	 */
 	if (more_entropy) {
-		uniqid = strpprintf(0, "%s%08x%05x%.8F", prefix, sec, usec, php_combined_lcg() * 10);
+		uint32_t bytes;
+		double seed;
+		if (php_random_bytes_silent(&bytes, sizeof(uint32_t)) == FAILURE) {
+			seed = php_combined_lcg() * 10;
+		} else {
+			seed = (10.0 / ((double) UINT32_MAX / bytes));
```

Comment:
I find the double division here rather odd, why not `((double) bytes / UINT32_MAX) * 10.0`?

It would be good if you could check how much this impact performance of uniqid. I'm not really concerned about the seeing for mt_rand as it happens once, but this happens on every call to uniqid.

Reply:
Fixed it. I'm so ashamed.

Trying to benchmarking `uniqid()`, There was no noticeable performance degradation, even in the most unlikely use cases. (In Debian Buster on Docker for Mac)

```
# original
$ time ./sapi/cli/php -r 'foreach (range(1, 100000) as $i) { uniqid(); }'

real    0m0.674s
user    0m0.071s
sys     0m0.595s

# use php_random_bytes()
$ time ./sapi/cli/php -r 'foreach (range(1, 100000) as $i) { uniqid(); }'

real    0m0.656s
user    0m0.073s
sys     0m0.572s
```

Reply:
The case you are testing does not have more_entropy set, so it does not actually test this code path. However, from my own test the new code is just 35% slower (Ubuntu), so I think this change is not problematic.